.github/workflows/test.yml

```diff
@@ -459,24 +420,13 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Setup Rust
-        uses: actions-rs/toolchain@v1
+        uses: ./.github/actions/setup-rust
+        with:
+          save-cache: true
 
       - name: Setup Node.js
         uses: ./.github/actions/setup-node
 
-      - name: Rust Cache
-        uses: Swatinem/rust-cache@v2
-
-      - name: Start Redis
-        uses: supercharge/redis-github-action@1.4.0
-        with:
-          redis-version: 6
-        if: matrix.os.name == 'ubuntu'
-
-      - name: Set Redis URL
-        run: echo "BULL_REDIS_CONNECTION=redis://127.0.0.1:6379" >> $GITHUB_ENV
-        if: matrix.os.name == 'ubuntu'
```

Comment:
Don't the node-file-trace tests need redis to work correctly?

Reply:
Oh, they do? I was looking for the `BULL_REDIS_CONNECTION` env var and couldn't find it anywhere

Reply:
tests pass without